sourcegraph/cody
# Edit: Improve response reliability with different chat models

## Description

This PR improves the `responseTransformer` (prev `contentSanitizer`) to better handle different responses from the LLM.

Specific changes:
- Better handling of XML tags, if the numbers differ from what we specified
- Markdown blocks are stripped (e.g. \`\`\`typescript foo \`\`\`)


## Test plan


1. Make Edits
2. Confirm that there's no regressions in output quality
